internal/events/eventstest/events_test_utils.go

```diff
@@ -0,0 +1,86 @@
+package eventstest
```

Comment:
@carbonin based on your comment on https://github.com/openshift/assisted-service/pull/1854#discussion_r687079898, WDYT ?

Reply:
Sorry for the late reply, we don't have a convention for this. 
IMO if we have something that is used by multiple tests we can have a separate package like here. 
we have `test_utils.go` in multiple packages and i don't like it. 
So having a separate model for this is actually a good idea.